modules/formulize/admin/config-sync.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
```

Comment:
So there's a semi standard boilerplate of comments that shows up at the top of most files. More recent ones are referencing the Formulize Project not Freeform Solutions, etc. What do you think about this in terms of the open source project? I really don't know much about the best practices here, but I feel like attaching some notice of the license and the project and copyright is a generally good idea.

Reply:
Most recent is I think modules/formulize/class/formulize.php which is by no means perfect, just a recent example